python/feathub/processors/flink/table_builder/tests/test_flink_processor.py

```diff
@@ -400,8 +402,6 @@ def setUp(self):
     def tearDown(self) -> None:
         MemoryOnlineStore.get_instance().reset()
         shutil.rmtree(self.temp_dir, ignore_errors=True)
-        KafkaSourceSinkITTest.tearDown(self)
```

Comment:
Would it be simpler to have a for-loop that enumerates all superclass and call `teardown()` for all superclasses who have `teardown()`?

If we agree to do this, we can either address it in this PR or add a TODO and work on this in a followup PR (without removing the code here).

Same for `setUp()` and for other processor IT tests.